workers.tf

```diff
@@ -1,20 +1,47 @@
-resource "aws_autoscaling_group" "workers" {
```

Comment:
This will cause the existing ASG to be destroyed.

Reply:
Fixed